app/scripts/controllers/decrypt-message.ts

```diff
@@ -45,7 +45,7 @@ export type CoreMessage = AbstractMessage & {
 };
 
 export type StateMessage = Required<
-  Omit<AbstractMessage, 'securityProviderResponse'>
```

Comment:
These are needed just to prevent TS complains. It will eventually get added with my PR here https://github.com/MetaMask/metamask-extension/pull/19184